### DISCUSSION TITLE
Error documentation

### DISCUSSION THREAD

**Discussion by EmmaTheMartian** [vlang/v#14641] [Ideas]
This is a suggestion to V's documentation, I have encountered multiple odd errors whilst working with V, and I've noticed that on the Discord for V, and it's issue tracker, the error I got was not found. The error wasn't even a compiler error.

Due to this, it would be beneficial to V to add documentation on some, or all, of the errors. Notably, the more strange errors or the ones that are more difficult to debug.

**Comment by JalonSolov** [comment]
If it's not known to the people on discord, and it's not already reported as an issue, please open a new issue yourself.

How are we supposed to document errors we don't know about?

**Comment by EmmaTheMartian** [comment]
That's a good point. Although, the "error documentation" is more so for errors that are in V, not builder errors. I.e, the more odd errors, like the one below, could have documentation to help the developers who use V.

This error in particular was a `Non constant array bound \`string\``. Despite it saying "array," it occurred whilst I was making a type alias for `map[string]string`.
